src/common/cmdparse.cc

```diff
@@ -600,8 +599,7 @@ template<bool is_vector,
 	 typename Value = std::conditional_t<is_vector,
 					     vector<T>,
 					     T>>
-bool validate_arg(CephContext* cct,
```

Comment:
ACK. `validate_arg()` is the leaf that was taking taking `cct` without any real need. Same applies to `validate_bool()`.